fetch.bs

```diff
@@ -5059,7 +5083,7 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
  <li><p>If <var>request</var>'s <a for=request>credentials mode</a> is not "<code>include</code>"
  and <var>origin</var> is `<code>*</code>`, then return success.
```

Comment:
```suggestion
  and <var>origin</var> is "<code>*</code>", then return success.
```

Reply:
In this case it's bytes, right? The value isn't decoded here.

Reply:
Ah yes, you're right.